contrib/drivers/lvm/targets/iscsi.go

```diff
@@ -21,48 +21,48 @@ import (
 	"strings"
 
 	log "github.com/golang/glog"
+	"github.com/opensds/opensds/pkg/utils"
+	"os"
 )
 
 type ISCSITarget interface {
-	CreateISCSITarget() error
-	GetISCSITarget() int
-	RemoveISCSITarget() error
+	CreateISCSITarget(volId, tgtIqn, path, initiator string, chapAuth []string) error
```

Comment:
I think it could be better if these parameters can be included in `tgtTarget` struct.